docs/conf.py

```diff
@@ -28,6 +28,10 @@
 cwd = os.getcwd()
 project_root = os.path.dirname(cwd)
 
+# On Travis python3-gi is installed into folllowing directory, which
+# is not added to path because Travis uses virtualenv.
+sys.path.insert(0, '/usr/lib/python3/dist-packages')
```

Comment:
Do we understand why `` system_site_packages: true`` does not enable access to the packed installed in the system dir?